classes/nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
```

Comment:
- [ ] Kindly add some error text to display. 
 Example:     `raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"`

Reply:
Thank you for being clear with your recommendation.